packages/esm-patient-list-management-app/src/lists-table/lists-table.component.tsx

```diff
@@ -268,17 +253,20 @@ const PatientListStarIcon: React.FC<PatientListStarIconProps> = ({ cohortUuid, i
   return (
     <TableCell className={`cds--table-column-menu ${styles.starButton}`} key={cohortUuid} style={{ cursor: 'pointer' }}>
       <Button
+        iconDescription="Star list"
         size={isTablet ? 'lg' : 'sm'}
         kind="ghost"
         hasIconOnly
         renderIcon={isStarred ? StarFilled : Star}
+        tooltipPosition="left"
```

Comment:
It makes more sense to position the tooltip to the left to avoid it getting truncated by the right edge of the page as is the case presently.